crates/sui-adapter-transactional-tests/tests/linters/self_transfer/self_transfer.exp

```diff
@@ -0,0 +1,6 @@
+processed 2 tasks
+
+task 1 'publish'. lines 3-30:
+created: object(1,0)
+mutated: object(0,0)
+gas summary: computation_cost: 1000000, storage_cost: 6870400,  storage_rebate: 0, non_refundable_storage_fee: 0
```

Comment:
@tnowacki the linter prints to stdout, but I don't see the output here. I'm guessing there's something in the transactional test adapter that hides all of the build output--do you have ideas on how to tweak this so we can exercise linters via transactional tests?

Reply:
Can you print to a buffer instead? Then you can pass up the buffer/string for the tests

Reply:
As far as I can tell this is not easily done unless @tnowacki is aware of some transactional testing magic that's not obvious to me... The problem is that this the linting diagnostics would have to somehow become part of publishing output, but publishing command in the transactional testing framework is handled on the "core" Move side of things (https://github.com/MystenLabs/sui/blob/25678de7f57438f7b9a6af8f908e3051d1505205/external-crates/move/testing-infra/transactional-test-runner/src/framework.rs#L604) and it does not seem to me at this point that it would be easy to rework this to include output from the linting pass...